pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -254,6 +255,28 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			return result, nil
 		}
 	}
+	// Handle InferenceService status updates based on the force stop annotation.
+	// If true, transition the service to a stopped and unready state; otherwise, ensure it's not marked as stopped.
+	if forceStopRuntime {
+		// Exit early if we have already set the status to stopped
+		existingStoppedCondition := isvc.Status.GetCondition(v1beta1.Stopped)
+		if existingStoppedCondition != nil && existingStoppedCondition.Status == corev1.ConditionTrue {
+			// TODO: Set condition to stoppING
```

Comment:
Is this a TODO for the future or did you mean to solve it in this PR?

Reply:
For the future, will be in next sprint